content/logs/_index.md

```diff
@@ -152,6 +152,8 @@ If the agent is containerized, see [the specific configuration file to mount the
 
 * [Restart your agent](https://help.datadoghq.com/hc/en-us/articles/203764515-Start-Stop-Restart-the-Datadog-Agent)
 
+## Log Agent advanced functionalities 
```

Comment:
Log Agent advanced usage
or 
Log Agent advanced functionality

Reply:
Thanks for the suggestion, we were struggling as hell on this one.

Log Agent advanced usage sounds perfect

Reply:
Also we probably dont want to call out Log Agent as users should experience Datadog as a single agent. So maybe its "Log collection" rather than "log agent"

Reply:
Make sens